default_python/requirements-dev.txt

```diff
@@ -3,6 +3,9 @@
 ## For defining dependencies used by jobs in Databricks Workflows, see
 ## https://docs.databricks.com/dev-tools/bundles/library-dependencies.html
 
+## Add code completion support for DLT
+databricks-dlt
```

Comment:
If we have this, would it be possible/desirable to enable `python.analysis.typeCheckingMode"`?